docs/sources/getting-started/get-started-grafana-influxdb.md

```diff
@@ -99,7 +99,8 @@ You can also create Flux queries in the InfluxDB Explore view.
 
 In your Grafana instance, go to the [Explore]({{< relref "../explore/" >}}) view and build queries to experiment with the metrics you want to monitor. Here you can also debug issues related to collecting metrics.
 
-#### Start building dashboards
+#### Start building dashboards or import from a curated list of existing InfluxDB dashboards.
```

Comment:
Is there a link we can add if folks want to import influxDB dashboards? This section title is too long. We might be able to change to "Start building or import dashboards" if we provide a link.